docs/api/topics/ratings.rst

```diff
@@ -21,6 +21,11 @@ _`List`
 
     :query app: the ID or slug of the app whose ratings are to be returned.
     :query user: the ID of the user or `mine` whose ratings are to be returned.
+    :query lang: a language to filter ratings by if `match_lang` is set.
+    :query match_lang: a boolean to specify to match langauge or not. If unset
+                       all results are returned, if 1 only results matching
+                       `lang` are returned if 0 only results not matching
+                       `lang` are returned. All other values are ignored.
```

Comment:
Just wondering if we could get away with a single value here, which is `lang`. If provided, we simply return only ratings matching the language. If not provide, we return all. I'm not sure I see a use-case for `match_lang=0`?